source/includes/reviews-for-all-apps.md

```diff
@@ -142,7 +142,7 @@ Field                    | Description
 **review_id**            | The ID assigned by the store. May be non-unique across apps.
 **review_text**          | The text of the body of the review.
 **title**                | The title/subject line of the review, as written by the user, if there is one.
-**user_id**              | The unique identifier for the user as assigned by the store ( deprecated for iTunes ). ![android_only](../images/android_logo.jpg)
+**user_id**              | The unique identifier for the user as assigned by the store ( deprecated for iTunes since August 2016 ). ![android_only](../images/android_logo.jpg)
```

Comment:
Probably want to add: "Only present in historical records"

Reply:
This is for reviews and field is present in our full dump for records before that date; adding this would create a bit of confusion imo.